content/influxdb/v1/about_the_project/release-notes.md

```diff
@@ -11,6 +11,173 @@ aliases:
+- Generalize test for Windows.
+- Use `copy` when a rename spans volumes.
+- Add tests for file rename across volumes.
+- Log errors in continuous query statistics storage.
+- Remove breaking argument validation for `_fieldKeys` iterator.
+- `ListBuckets` properly returns a wrapped response.
+- Don't write skipped shard messages to the line protocol output destination.
+- Add warning if `fields.idxl` is encountered.
+- Prevent world-writable `MANIFEST` files.
+  to compact series files at startup.
+- Do not escape CSV output.
+- Avoid SIGBUS when reading non-standard series segment files.
+- Prevent retention service from creating orphaned shard files.
+- Sync index file before closing.
+- Abort processing a write request when encountering a precision error.
+- `MeasurementsCardinality` should not be less than 0.
+- Replace unprintable and invalid characters in errors.
+- Keep TSI reference or don't close `TagValueSeriesIDIterator` on error.
+- Eliminate race condition on `Monitor.globalTags`.
+- Correctly handle `MaxSeriesPerDatabaseExceeded`.
+- Return underlying errors when creating a subscription.
+- Fix a race condition that caused restore commands to fail.
+- Create shards without overlapping time ranges.
+- Improve error messages for snapshots with no data.
+- Detect misquoted tag values and return an error.
+- Use `debug` log level in `influx_tools compact-shard`.
+- Allow limiting shards during Flux iterator reads.
+- When restoring data, force the `-db` parameter when `-newdb` is used.
+- Sync series segments to disk after writing.
+- Fix deadlock in `influx_inspect dumptsi`.
+- Prevent errors when restoring from a portable backup.
+- Copy names from mmapped memory before closing the iterator.
+- TSI should compact log files that are old or too large.
+- Compact old `tsl` files without new writes.
+- Enforce a hard limit on field size while parsing line protocol.
+- Ensure log formatting (JSON) is respected.
+- Ensure systemd handles HTTPS and 40x response codes and blocks indefinitely
+  while waiting for API repsonses.
+- Avoid compaction queue stats flutter.
+- Require database authorization to see continuous queries.
+- Return correct count of `ErrNotExecuted`.
+- Address various `staticcheck` warnings.
+- Properly shutdown multiple HTTP servers.
+- Make error messaging for missing shards consistent.
+- Fix redundant registrations for prometheus collector metrics.
+- Improved performance for sorted merge iterator.
+- Grow tag index buffer when needed.
+- Fixes key collisions when serializing `/debug/vars`.
+- Return an error when trying to delete a nonexistent shard ID.
+- Improved CORS support:
+  - Add User-Agent to allowed CORS headers.
+  - Allow CORS in v2 compatibility endpoints.
+  - Allow the `PATCH` request method.
+
+### Other
+
+- Upgrade to Go 1.20.13.
+- Pin Rust version to 1.52.1.
+- Upgrade Flux to v0.194.4.
+- Upgrade multiple dependencies for maintenance and security hardening.
```

Comment:
What security hardening were you thinking of? (I can't remember and the changelog isn't reminding me)

Reply:
I just grouped many of the dependency update commits into this one note. Here's the specific commit that references security: https://github.com/influxdata/influxdb/commit/90c15ff

Reply:
It's fine as is. 'hardening' made my think of the 2.x hardening options that were added that we didn't backport to 1.x is all. Feel free to leave as is or drop `and security hardening`.